server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -544,6 +550,35 @@ private static WebAuthnPolicy getWebAuthnPolicyPasswordless(RealmRepresentation
         return webAuthnPolicy;
     }
 
+    private static CIBAPolicy convertCIBARepresentationToPolicy(CIBARepresentation cibaRep) {
+        CIBAPolicy cibaPolicy = new CIBAPolicy();
+        if (cibaRep != null) {
+            String cibaBackchannelTokenDeliveryMode = cibaRep.getCibaBackchannelTokenDeliveryMode();
```

Comment:
This should remove some boilerplate code:

```
cibaPolicy.setBackchannelTokenDeliveryMode(Optional.ofNullable(cibaRep.getCibaBackchannelTokenDeliveryMode())
                    .filter(StringUtil::isNotBlank)
                    .orElse(Constants.DEFAULT_CIBA_POLICY_TOKEN_DELIVERY_MODE));
```

Reply:
@pedroigor I see, I will fix codes as you proposed instead of current boilerplate style codes.